Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -61,21 +61,18 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 PercExcl = NaN;
 MinimumtValue = NaN;
 
-if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE
+if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE || x.modules.asl.bContainsDeltaM
     % ENABLE is disabled if multiPLD/TE

```

Comment:
I would move `x.modules.asl.bContainsDeltaM` to a separate if-statement at the beginning of `xASL_wrp_RealignASL`, because in this case we want to skip motion correction completely right?

Reply:
The input parameter `bSubtraction` was confusing as it wasn't clear if this refers to the fact that input is 'already subtracted' or 'subtraction-based images'. I have changed that to `bASL` is it is literally only ASL that has this option true. And it is only an internal parameter. This way, we won't confuse it with the `bContainsDeltaM` parameter.

To your question: It's a bit more complex.
- bContainsDeltaM turns off ENABLE and ZigZag as both need control/labels currently
- multi-PLD turns of both as well as with ENABLE it will be really tricky to exclude something, and ZigZag needs to be specially adapted for multi-PLD - but that's temporarily.
- However, single-PLD and bContainsDeltaM with multiple averages (though not sure if we have such a flavor) can run motion correction with ENABLE=0 and ZigZag = 0.

Finally, bZigZag was ignored currently in the code and was assigned 'manually'. I have fixed that as well.

All in 3d240036

Reply:
See below.